### DISCUSSION TITLE
[Contribution] What is the main problem to solve around emerging UX guidance?

### DISCUSSION THREAD

**Discussion by selenehinkley** [Shopify/polaris-react#6435] [General]
The Enablement blobby of the Polaris team is trying to land on the key problem we want to solve around documentation of emerging UX guidance. Our initial attempt was to have an **Enable publishing of emerging guidance project** under the following guiding statement:

```Shopifolk feel enabled and encouraged to contribute because the process is clear, easy and fast.```

We quickly realized this was not a quick, 3 week project and that we couldn't clearly define what "enable" meant in this context. We've have decided to spend some time thinking about the true problem we want to solve, and then identify the format and timely that the project will take.

Here is an important question we posed during our conversation: How can we empower people to feel confident having their guidance opensource? (and not jump to internal). 

Posting this in a github discussion because we want to continue this conversation async until early next week (first week of July 2022).

**Comment by selenehinkley** [comment]
Yesenia offered some helpful input during our 1:1 today:

Start with examples we see today: identify, assess, reach out to collaborate, contribute
Examples that come to mind: [How to ask users for feedback](https://shopify.slack.com/archives/C0183TTG88P/p1656412983802649). 

Rough steps in this process:
1. Spend intentional time looking for conversations around emerging guidance
2. Play detective and reach out to the crafters who have the most context/impact on the problem being solved
4. Guide folks in this process. Help them determine what type of guidance to offer and where it lives. The first round won't be as self serve and that's okay.
5. Help them publish the guidance in areas that exist today
6. Start documenting pain points, length of support, discipline type, quality, etc
7. When we feel caught up, do a retrospective and identify what can be improved next

Additional notes:
- Thumbs up to trying to keep it opensource.

Based on this guidance, the problems we're initially solving are:

- The current version of Polaris.Shopify.com doesn't reflect guidance that exists today
- Since the guidance doesn't have visibility, crafters don't know it exists and create solutions in silos (which is an inefficient way of working that creates a more disjointed merchant experience)

Goal: operationalize as an Enablement blobby to reduce the barriers that are causing these problems, even if V1 involves more manual support on our end.

**Reply by yurm04** [reply]
@selenehinkley If i'm understanding @yesenia-perezcruz 's guidance this first round of the project is more hands on and investigative:
Spend some time finding conversations with emerging guidance and partner with those teams to add that guidance to the patterns (?) guidance.

From this first more investigational and hands-on project we can come up with "operationalized" guidance for teams to contribute emerging guidance independently without our intervention.

Do I have this right?
